### PR TITLE
Add function cider-insert-region-in-repl

### PR DESCRIPTION
**Pull request by syl20bnr**
Hi Bozhidar,

This PR adds a function to send the current region to the REPL.

Cheers,
syl20bnr

### CODE REVIEW

**Comment by bbatsov** [comment]
You might mention this in the README as well.

File: cider-interaction.el [hunk 1]
```diff
@@ -1496,6 +1496,14 @@ If invoked with a prefix ARG eval the expression after inserting it."
   (interactive "P")
   (cider-insert-in-repl (cider-defun-at-point) arg))
 
+(defun cider-insert-region-in-repl (&optional arg)
+  "Insert the curent region in the REPL buffer.
+If invoked with a prefix ARG eval the expression after inserting it."
+  (interactive "P")
+  (cider-insert-in-repl
+   (buffer-substring-no-properties (region-beginning)
```
Comment (bbatsov) [inline cider-interaction.el]: Btw, why don't you use `r` in `interactive`?

**Comment by syl20bnr** [comment]
Thank you, I updated to use "r".

For the readme update, I don't see where I can add it, there is only a key binding for `cider-insert-last-sexp-in-repl` and not for the other related `insert` commands and there is no mention of these commands anywhere else.

**Comment by bbatsov** [comment]
Ops, I meant to say the changelog, not the readme.

**Comment by syl20bnr** [comment]
Ok :-)
Done!

**Comment by bbatsov** [comment]
:+1:

**Pull request merged**